build.sbt

```diff
@@ -18,26 +18,25 @@ import sbt._
 import sbt.Keys._
 import uk.gov.hmrc.DefaultBuildSettings.targetJvm
 import uk.gov.hmrc.SbtAutoBuildPlugin
-import uk.gov.hmrc.SbtArtifactory
 import uk.gov.hmrc.versioning.SbtGitVersioning.autoImport.majorVersion
 import uk.gov.hmrc.versioning.SbtGitVersioning
 
 lazy val library = (project in file("."))
-  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
+  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning)
   .settings(PlayCrossCompilation.playCrossCompilationSettings)
   .settings(
-    scalaVersion := "2.12.12",
+    scalaVersion := "2.12.14",
     name := "play-json-union-formatter",
     majorVersion := 1,
-    makePublicallyAvailableOnBintray := true,
+    isPublicArtefact := true,
     targetJvm := "jvm-1.8",
     libraryDependencies ++= deps
   )
 
 val testDepsShared: Seq[ModuleID] = Seq(
-  "org.scalatest" %% "scalatest" % "3.0.8" % "test",
-  "org.pegdown"   %  "pegdown"   % "1.6.0" % "test",
```

Comment:
`sbt-settings` turns on HTML reports [here](https://github.com/hmrc/sbt-settings/blob/master/project/DefaultBuildSettings.scala#L48) so we must use the HTML report library that ScalaTest currently uses